ngi_pipeline/conductor/launchers.py

```diff
@@ -66,7 +76,31 @@ def launch_analysis(projects_to_analyze, restart_failed_jobs=False,
         except (RuntimeError, KeyError, CharonError) as e: # BPA missing from Charon?
             LOG.error('Skipping project "{}" because of error: {}'.format(project, e))
             continue
+        try:
+            qc_analysis_module = load_engine_module("qc")
```

Comment:
@mariogiov here I can see you define `qc_analysis_module` and a couple of lines down here your comment is `# launch QC analysis` but no reference to `qc_analysis_module` is present.... is this intentional? no it is not intentional I am pretty sure that 
```analysis_module.analyze(project=project,
                                       sample=sample,
                                       config=config)
```
needs to be 

```qc_analysis_module.analyze(project=project,
                                       sample=sample,
                                       config=config)
```